sbi/inference/potentials/likelihood_based_potential.py

```diff
@@ -115,6 +116,38 @@ def __call__(self, theta: Tensor, track_gradients: bool = True) -> Tensor:
                 )
             return log_likelihood_batches + self.prior.log_prob(theta)  # type: ignore
 
+    def condition_on(self, condition: Tensor, dims_to_sample: List[int]) -> Callable:
+        """Returns a potential conditioned on a subset of theta dimensions.
+
+        The condition is a part of theta, but is assumed to correspond to a batch of iid
+        x_o.
+
+        Args:
+            condition: The condition to fix.
```

Comment:
Should the condition be the full `theta` or only the ones that are not part of `dims_to_sample`?

Reply:
The sentence above implies that it is only "a part of theta"